__output__/cosmwasm/CW4GroupContract.react-query.ts

```diff
@@ -30,7 +30,7 @@ export function useCW4GroupMemberQuery({
   args,
   options
 }: CW4GroupMemberQuery) {
-  return useQuery<MemberResponse, Error, MemberResponse, (string | undefined)[]>(["cW4GroupMember", client.contractAddress], () => client.member({
+  return useQuery<MemberResponse, Error, MemberResponse, (string | undefined)[]>(["cW4GroupMember", client.contractAddress, JSON.stringify(args)], () => client.member({
     addr: args.addr,
```

Comment:
what is the `JSON.stringify(args)` for?

Reply:
This is actually from https://github.com/CosmWasm/ts-codegen/pull/35#issuecomment-1203194829, which I'll now rebase.